aspnet/web-api/overview/advanced/calling-a-web-api-from-a-net-client.md

```diff
@@ -158,7 +158,7 @@ Like GET, a DELETE request does not have a request body. You don't need to speci
 
 To test the client app:
 
-1. [Download](https://github.com/aspnet/Docs/tree/master/aspnet/web-api/overview/advanced/calling-a-web-api-from-a-net-client/sample/server) and run the server app. [Download instructions](/aspnet/core/tutorials/#how-to-download-a-sample). Verify the server app is working. For example, `http://localhost:64195/api/products` should return a list of products.
+1. [Download](https://github.com/aspnet/AspNetDocs/tree/master/aspnet/web-api/overview/advanced/calling-a-web-api-from-a-net-client/sample/server) and run the server app. [Download instructions](/aspnet/core/tutorials/#how-to-download-a-sample). Verify the server app is working. For exaxmple, `http://localhost:64195/api/products` should return a list of products.
```

Comment:
Looks like this reverted the change

Reply:
Thank you for pointing that out. I've fixed it with the following commit: https://github.com/aspnet/AspNetDocs/commit/e221d2f9a1759f131e6db2866b8bfb5fb211b6ac